pages/dao/[symbol]/proposal/components/instructions/Vsr/VotingMintConfig.tsx

```diff
@@ -192,17 +264,25 @@ const VotingMintConfig = ({
           }
         }
       ),
+    ...(!showHeliumFields
```

Comment:
nit: i think its weird how the schema lives inside this component. thats not your fault but would it be possible to make this check simply look at the form contents, rather than depending on external state? and then maybe just extract this schema? or at least memoize it.